pkg/virt-api/api.go

```diff
@@ -375,10 +374,10 @@ func (app *virtAPIApp) composeSubresources() {
 
 				response.WriteAsJson(list)
 			}).
-			Operation("getAPIResources").
+			Operation("getAPISubResources").
```

Comment:
what does changing the operation name here impact?

Reply:
I believe that impact is only in the documentation. I changed it because kube-openapi demands unique operations...